lib/engine/game/base.rb

```diff
@@ -764,6 +767,35 @@ def last_game_action_id
         @last_game_action_id || 0
       end
 
+      def previous_game_action_id_from(action_id)
```

Comment:
rename all of these to be without the game, so previos_action_id(action)

Reply:
I feel like previous_action_id is misleading as this will return the previously processed player move, ignoring any messages and undone actions in between the current move and the last move, so it isn't the previous action. I worry without adding another descriptive word, the method will be misleading to others. That's why I added the word "game" to the method name. Thoughts?

Reply:
the game in it doesn't help that goal imo, it just makes it more verbose.